config/routes.rb

```diff
@@ -2,6 +2,9 @@
   devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
+  devise_scope :user do
+    get '/users/sign_out' => 'devise/sessions#destroy'
+  end
```

Comment:
We could revise this part after registration functionality is created, because currently or application doesn't allow any user to sign up.